Utils/Dataflow/071_esConsistency/consistency.py

```diff
@@ -160,19 +160,21 @@ def process(stage, message):
     # It's unlikely that such documents will be produced in DKB. In general,
     # such documents should be checked by es.exists(), and not es.get().
     if not data:
-        log('Nothing to check for document (%s, %d)' % (_type, _id), 'WARN')
+        log('Nothing to check for document ({0}, {1})'.format(_type, _id),
+            'WARN')
```

Comment:
It doesn\`t make much change if `_id` is string or what; integer still can be output with `%s`:
```
>>> print "Integer: %s" % 12345
Integer: 12345
>>> print "String : %s" % '12345'
String : 12345
>>> print "Integer: {}".format(12345)
Integer: 12345
>>> print "String : {}".format('12345')
String : 12345
```

It is better, though, to output such things with `%r` to make the difference:
```
>>> print "Integer: %r" % 12345
Integer: 12345
>>> print "String : %r" % '12345'
String : '12345'
```
Yet I almost never remember to think about it writing my code... )

---
*This is just a comment: there\`s nothing bad in `.format()`, in fact. We rarely use it in the code, that\`s true, but it\`s a question of a habit. Aometimes I even think that maybe we should prefer it as a more modern form... and then forget about it again ;)*

Reply:
Thanks for telling me about `%r`, it looks useful. Changed the formatting to use `%r`.